### ISSUE TITLE
'Inline variable declaration' spacing incorrect with csharp_space_around_declaration_statements=ignore

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#35993]
VSF_TYPE_MARKDOWNWhen I run CodeCleanup (EditorContextMenus.FileHealthIndicator.RunDefaultCodeCleanup) and it replaces predefined types w/Framework types the result is sometimes erroneous - when the substitution is on a method-call with 'out' parameters the space between the type-name and the variable is sometimes dropped (e.g. x(out int i1) becomes x(out Integer32i1). This only seems to happen when the method-call is part of an assignment-statement.

In addition to attaching my .editorconfig I've also attached the following:
original.png -sample showing original code
AssignFirst.png -shows erroneous results when the method-call is part of an assignment
AssignSecond.png -shows correct results when method-call is stand-alone

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/583507/codecleanup-replace-predefined-type-wframework-som.html
VSTS ticketId: 899753_
_These are the original issue comments:_

Lee Silver on 5/26/2019, 09:00 PM (33 hours ago): In my original report I wrote "(e.g. x(out int i1) becomes x(out Integer32i1)" when it should have been "(e.g. x(out int i1) becomes x(out Int32i1)"
Visual Studio Feedback System on 5/27/2019, 05:10 AM (25 hours ago): <p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

Lee Silver on 5/27/2019, 05:19 PM (13 hours ago): I just noticed that during CodeCleanup 'out'-variable preferences are done after 'type' preferences. Since the 'type' has the correct substituted value it looks like the problem is with the handling of 'out' variable preferences.
Visual Studio Feedback System on 5/28/2019, 03:58 AM (2 hours ago): <p>This issue is currently being investigated. Our team will get back to you if either more information is needed, a workaround is available, or the issue is resolved.</p>

_These are the original issue solutions:_
(no solutions)

**Comment by sharwell** [comment]
Source code:

```csharp
using System;

namespace ClassLibrary5
{
    public class Class1
    {
        void A()
        {
            bool x;
            var result = B(out x);
        }

        object B(out bool x)
        {
            x = default;
            return default;
        }
    }
}
```

.editorconfig

```ini
[*.cs]
dotnet_style_predefined_type_for_locals_parameters_members = false:silent
csharp_style_var_for_built_in_types = false:silent
csharp_space_around_declaration_statements = ignore
```

Steps to reproduce:

1. Create a C# project with the above source file and .editorconfig file
2. Apply the **Inline variable declaration** code fix on `bool x` on line 9

Expected result:

```csharp
using System;

namespace ClassLibrary5
{
    public class Class1
    {
        void A()
        {
            var result = B(out Boolean x);
        }

        object B(out bool x)
        {
            x = default;
            return default;
        }
    }
}
```

Actual result:

```csharp
using System;

namespace ClassLibrary5
{
    public class Class1
    {
        void A()
        {
            var result = B(out Booleanx);
        }

        object B(out bool x)
        {
            x = default;
            return default;
        }
    }
}
```

**Comment by LeeMSilver** [comment]
Thanks for the quick diagnosis of the problem - the problem is not what I thought is would be.

I suspect that a fix in 16.1 that affected 'csharp_space_around_declaration_statements=ignore' caused this problem to arise. That fix had to do with spaces in declaration statements NOT being ignored.

**Comment by LeeMSilver** [comment]
In my last comment I forgot to mention that the fix I was referring to is #31868.
